js/passwordManager/bitwarden.js

```diff
@@ -161,41 +153,43 @@ class Bitwarden {
       console.warn(e)
     }
 
-    // show credentials dialog
-
-    var signInFields = [
+    // show ask-for-credential dialog
+    const signInFields = [
+      { placeholder: 'Server URL (Leave blank for the default Bitwarden server)', id: 'url', type: 'text' },
       { placeholder: 'Client ID', id: 'clientID', type: 'password' },
       { placeholder: 'Client Secret', id: 'clientSecret', type: 'password' }
     ]
 
-    const credentials = ipcRenderer.sendSync('prompt', {
-      text: l('passwordManagerBitwardenSignIn'),
-      values: signInFields,
-      ok: l('dialogConfirmButton'),
-      cancel: l('dialogSkipButton'),
-      width: 500,
-      height: 260
-    })
-
-    for (const key in credentials) {
-      if (credentials[key] === '') {
-        throw new Error('no credentials entered')
+    const credentials = ipcRenderer.sendSync(
+      'prompt',
+      {
+        text: l('passwordManagerBitwardenSignIn'),
+        values: signInFields,
+        ok: l('dialogConfirmButton'),
+        cancel: l('dialogSkipButton'),
+        width: 500,
+        height: 260
```

Comment:
The dialog is a little bit too short now, at least on macOS:

<img width="884"  src="https://github.com/minbrowser/min/assets/10314059/de798ffa-c106-4845-98ec-8d4d3b00b715">